dbt_copilot_helper/commands/copilot.py

```diff
@@ -186,6 +194,11 @@ def make_addons(directory="."):
     _cleanup_old_files(config, output_dir, env_addons_path)
     custom_resources = _get_custom_resources()
 
+    svc_names = list_copilot_local_services()
+    base_path = Path(directory)
+    for svc_name in svc_names:
+        _generate_svc_overrides(base_path, templates, svc_name)
```

Comment:
Could this be called from `_generate_service_addons()`?

Reply:
I did look at that option, but I think we will miss some services, unless they happen to be defined under an addon in addons.yml. e.g.

```
dw-s3-bucket:
  type: s3
  services:
    - "s3proxy"
    - "beat"
    - "web"
    - "worker"
```
Not all applications will have a setup like DW

Reply:
Oh yeah, I think I slightly misunderstood what was going on down below.